google/devtools/clouderrorreporting/v1beta1/clouderrorreporting_gapic.yaml

```diff
@@ -0,0 +1,177 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouderrorreporting.spi.v1beta1
+  python:
+    package_name: google.cloud.devtools.clouderrorreporting.v1beta1
+  go:
+    package_name: cloud.google.com/go/devtools/clouderrorreporting/apiv1beta1
+  csharp:
+    package_name: Google.Devtools.Clouderrorreporting.V1beta1
+  ruby:
+    package_name: Google::Devtools::Clouderrorreporting::V1beta1
+  php:
+    package_name: Google\Cloud\Devtools\Clouderrorreporting\V1beta1
+interfaces:
+- name: google.devtools.clouderrorreporting.v1beta1.ErrorGroupService
+  collections:
+  - name_pattern: projects/{project}/groups/{group}
+    entity_name: group
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 20000
+    rpc_timeout_multiplier: 1
```

Comment:
This looks a bit weird to me. Does it mean that the rpc timeout won't be extended since the multiplier is 1?